src/sentry/static/sentry/app/views/performance/vitalsCards.tsx

```diff
@@ -37,12 +38,35 @@ type Props = {
   showVitalPercentNames?: boolean;
   showDurationDetail?: boolean;
   hasCondensedVitals?: boolean;
+  projects: Project[];
 };
 
-export default function VitalsCards(props: Props) {
-  const {eventView, organization, location} = props;
+// Temporary list of platforms to only show web vitals for.
+const VITALS_PLATFORMS = [
+  'javascript',
+  'javascript-react',
+  'javascript-angular',
+  'javascript-angularjs',
+  'javascript-backbone',
+  'javascript-ember',
+  'javascript-gatsby',
+  'javascript-vue',
+];
+
+function VitalsCards(props: Props) {
+  const {eventView, organization, location, projects} = props;
   const vitalsView = eventView.clone();
 
+  const showVitalsCard = vitalsView.project.some(projectId =>
```

Comment:
The performance of this might look a little slow but I think real world performance will be fine considering we have low `n` and it has an early exit.

Reply:
The only potential optimization I can think of is to convert `projects` into a map of project:platform first so that you don't have to repeat `.find()` n times.